reana_server/reana_admin/check_workflows.py

```diff
@@ -146,73 +164,144 @@ def check_workflows(
 
     time_range_message = f"{date_start} - {date_end if date_end else 'now'}"
 
-    if filtered_workflows.count() == 0:
+    total_workflows = filtered_workflows.count()
+
+    if total_workflows == 0:
         click.secho(
-            f"No workflows found matching time range {time_range_message}.\nPlease, adjust your filter conditions.",
+            f"No workflows found matching time range {time_range_message}.",
             fg="red",
         )
-        sys.exit(1)
+        return [], [], total_workflows
     click.secho(
-        f"Found {filtered_workflows.count()} workflow(s) matching time range {time_range_message}.\n",
+        f"Found {total_workflows} workflow(s) matching time range {time_range_message}.",
     )
 
     try:
         click.secho("Collecting MQ messages...")
         scheduler_messages = _collect_messages_from_scheduler_queue(filtered_workflows)
     except Exception as error:
-        click.secho(
-            "Error is raised when collecting scheduler messages.",
-            fg="red",
-        )
         logging.exception(error)
```

Comment:
What about moving all the `logging.exception(...)` in `check_workflows` and `check_interactive_sessions` to the main command function, where we are already catching and printing this exceptions? WDYT?

Reply:
I removed click.secho in the `exception` clause as it is not helpful. Now, it will just log an exception that has traceback and message.